pkg/controller/proxyconfig/controller.go

```diff
@@ -154,3 +154,9 @@ func isSpecHTTPSProxySet(proxyConfig *configv1.ProxySpec) bool {
 func isSpecNoProxySet(proxyConfig *configv1.ProxySpec) bool {
 	return len(proxyConfig.NoProxy) > 0
 }
+
+// isSpecReadinessEndpoints returns true if spec.readinessEndpoints of
+// proxyConfig is set.
+func isSpecReadinessEndpoints(proxyConfig *configv1.ProxySpec) bool {
```

Comment:
i might be stepping on a golang idiom, but i would have thought to call this "has" not "is"

Reply:
Thanks for catching this. The func name should have `Set` suffix.